src/settings-web/src/components/GeneralSettings.tsx

```diff
@@ -121,11 +121,14 @@ export class GeneralSettings extends React.Component <any, any> {
         }
         <Separator />
         <Text variant='xLarge'>General</Text>
-        <BoolToggleSettingsControl
-          setting={{display_name: 'Run at Startup', value: this.state.settings.general.startup}}
-          on_change={this.parent_on_change}
-          ref={(input) => {this.startup_reference=input;}}
-        />
+        {this.state.settings.general.startup != null &&
```

Comment:
I think we should pass a "running as packaged" setting. It would be more readable.

Reply:
@bzoz you mean renaming `startup` to e.g. `running_as_packaged`?

Reply:
No, the startup should stay. I meant adding a new, separate flag

Reply:
ah, sorry that makes sense :)